docs/openapi/components/schemas/common/OrganicInspection.yml

```
@@ -0,0 +1,279 @@
+  continuingCertification:
+    title: Continuing Certification
+    description: The applicant was previously organic certified.
+    type: boolean
+    $linkedData:
+      term: continuingCertification
+      '@id': https://service.unece.org/trade/uncefact/vocabulary/uncefact/#information
+  newLocationActivity:
+    title: New Production Location or Activity
+    description: (If continuing certification) the applicant was previously organic certified and has since added a new production location or activity.
+    type: boolean
+    $linkedData:
+      term: newLocationActivity
+      '@id': https://service.unece.org/trade/uncefact/vocabulary/uncefact/#information
+  reinstatement:
+    title: Reinstatement
+    description: The applicant was, but is no longer, organic certified.
+    type: boolean
+    $linkedData:
+      term: reinstatement
+      '@id': https://service.unece.org/trade/uncefact/vocabulary/uncefact/#information
+  announcedInspection:
+    title: Announced Inspection
+    description: The inspection was announced, as opposed to an unnanounced inspection.
+    type: boolean
+    $linkedData:
+      term: announcedInspection
+      '@id': https://service.unece.org/trade/uncefact/vocabulary/uncefact/#information
+  estimatedHarvestDate:
+    title: Estimated Harvest Date
+    description: The estimated harvest date for the applicant's operation.
+    type: string
+    $linkedData:
+      term: estimatedHarvestDate
+      '@id': https://www.gs1.org/voc/harvestDate
+  pesticideResidueSampling:
+    title: Pesticide Residue Sampling
+    description: Pesticide residue sampling was conducted.
+    type: boolean
+    $linkedData:
+      term: pesticideResidueSampling
+      '@id': https://service.unece.org/trade/uncefact/vocabulary/uncefact/#information
+  samplingDetails:
+    title: Sampling Details
+    description: If pesticide residue sampling was conducted, please provide details.
+    type: string
+    $linkedData:
+      term: samplingDetails
+      '@id': https://service.unece.org/trade/uncefact/vocabulary/uncefact/#content
+  introductionOperationDescription:
+    title: Introduction / Operation Description
+    description: Describe conditions at the time of inspection, and any general summary comments about the operation needed to complement the description in the client's OSP. Note to which standards (NOP) compliance is verified, including any additional certifications (international standards, GAPs, etc).
+    type: string
+    $linkedData:
+      term: introductionOperationDescription
+      '@id': https://service.unece.org/trade/uncefact/vocabulary/uncefact/#description
+  resolutionIssuesActionItems:
+    title: Resolution of Issues and Action Items
+    description: List any previously identified issue, non-compliance, corrective action required, or request for further information provided by the Certifier, a description of how it has been addressed; whether is it resolved, or is pending further action.
```

```suggestion
    description: List any previously identified issue, non-compliance, corrective action required, or request for further information provided by the Certifier, a description of how it has been addressed, i.e., whether is it resolved, or is pending further action.
```

---

"whether it is resolved, or is pending further action" is not a summary of the previous statement, it's an additional thing to note in that field.

Whether a semicolon is technically correct for this is debatable, but a) it gets the point across and is appropriate for terse delivery, and b) I'm copying all of this directly from the source material.

---

Regrettably, plenty of the source material includes language issues/errors.  Where we can do better, I think we should.

Now I *think* I understand what they meant to say, here's a revised suggestion --
```suggestion
    description: List any previously identified issue, non-compliance, corrective action required, or request for further information provided by the Certifier; a description of how each has been addressed; and whether each has been resolved or is pending further action.
```